astropybotrules.yaml

```diff
@@ -0,0 +1,2 @@
+CHANGELOG_CHECK: false
+AUTOCLOSE_STALE_PULL_REQUEST: false
```

Comment:
We should probably make the default name ``.astropybot.yml``

Reply:
Is a hidden file actually beneficial? Appveyor's not hidden, while Travis' is, so people are designing it either way, and there is no clear "winner".

Reply:
`astropybotrules` is a word play on "Cider House Rules" but it is pretty weak, so 👍 for `.astropybot.yml` (whether with leading period or not).

Reply:
I would go for hidden - I like the pun, but I also would prefer that we avoid inside jokes for files that will be in the main repo(s). But nothing to say you can't make puns/jokes inside the bot code!

Reply:
I for one also prefer the hidden ones, and appveyor.yml thus annoys me greatly...